frontend/src/metabase/admin/app/reducers.ts

```diff
@@ -82,14 +79,6 @@ const paths = createReducer(getAdminPaths(), (builder) => {
   });
 });
 
-const isNoticeEnabled = createReducer(
-  Settings.deprecationNoticeEnabled(),
-  (builder) => {
-    builder.addCase(disableNotice.fulfilled, () => false);
```

Comment:
we don't need this reducer anymore since we're not persisting this as independent state, it's purely derived from admin settings, which get refreshed if we set the disable notice to the current version